translate/filters/checks.py

```diff
@@ -55,14 +55,16 @@
         %(                          # initial %
         (?P<boost_ord>\d+)%         # boost::format style variable order, like %1%
         |
+        \|?                         # boost::format allows to enclose params in pipes, %|spec|
               (?:(?P<ord>\d+)\$|    # variable order, like %1$s
               \((?P<key>\w+)\))?    # Python style variables, like %(var)s
         (?P<fullvar>
-            [+#-]*                  # flags
+            [+#\- ]*                # flags
```

Comment:
Ok. I wonder now if we should make this flexible by using `\s` instead of ` ` (whitespace).